File: src/main/java/com/ifat/bdd/spark/labs/lab_taxi_lab/mainly_java_form/model/Trip.java
package com.ifat.bdd.spark.labs.lab_taxi_lab.mainly_java_form.model;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;

import java.time.LocalDateTime;

@Builder
@Data
@Getter
public class Trip {
    private String driverId;
    private String tripTargetCity;
    private int tripLength;
    private LocalDateTime tripDateTime;
}

File: src/main/java/com/ifat/bdd/spark/labs/lab_taxi_lab/mainly_scala_form/model/Trip.scala
package com.ifat.bdd.spark.labs.lab_taxi_lab.mainly_scala_form.model

import java.time.LocalDate

case class Trip( driverId: String,  tripTargetCity: String,  tripLength: Int,  tripDateTime: LocalDate) {
  println(s"driverId=$driverId tripTargetCity=$tripTargetCity tripLength=$tripLength date=$tripDateTime ")
}
